lib/pure/collections/critbits.nim

```diff
@@ -237,7 +237,8 @@ iterator mpairs*[T](c: var CritBitTree[T]): tuple[key: string, val: var T] =
   ## yields all (key, value)-pairs of `c`. The yielded values can be modified.
   for x in leaves(c.root): yield (x.key, x.val)
 
-proc allprefixedAux[T](c: CritBitTree[T], key: string; longestMatch: bool): Node[T] =
+proc allprefixedAux[T](c: CritBitTree[T], key: string;
+    longestMatch: bool): Node[T] =
```

Comment:
These splits look a little strange, and are inconsistent with the rest of the splits in this file. Proper alignment imo would be to match up the argument with the previous arguments, like so:
```nim
proc allprefixedAux[T](c: CritBitTree[T], key: string;
                       longestMatch: bool): Node[T] =
```

Reply:
Done.

Reply:
Same thing on line 214